force-app/main/default/classes/PMT_RefundController_TEST.cls

```diff
@@ -48,6 +48,36 @@ public with sharing class PMT_RefundController_TEST {
         }
     }
 
+    @IsTest
+    private static void verifyCalloutIsMadToRetrieveElevateRefundInformation() {
```

Comment:
should add "e" to "Mad" - should be "verifyCalloutIsMadeToRetrieveElevateRefundInformation"